aries-site/src/pages/whats-new.mdx

```diff
@@ -52,14 +52,87 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 4 - January 21, 2022
+
+### Roadmap Updates
+- **Tag** - Completed guidance for [Tag](https://design-system.hpe.design/components/tag).
+- **Navigation** 
+   - Completed research for [Ascending Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827).
+   - Completed ideations for [Ascending Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2725%3A18556).
+- **Button** - [Updated font color on default button active states](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component) from default text to text-strong.
+- **Page Layouts** 
+   - Completed Dashboard template for a [2 column layout](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2231%3A17992).
+   - Shared status update with the Design System Council on January 20, 2022.
+   - Completed documentation around the definition, scope and anatomy of Page Layouts.
+   - Created guidelines for the Page Container, including defining 3 types of content widths: Wide, Narrow and Full.
+   - Defined new breakpoints and page margins for each breakpoint.
+   - Work in progress for Page Layout templates, starting with 2 column, 3 column and multi-column Dashboards.
+
+### Stay current with the latest changes
+- **Voice and Tone** - Updates in progress for Voice & Tone page.
```

Comment:
For respect of our readers' time, I suggest we only include tangible, milestone updates. "In progress" feels not exceptionally helpful. I'd either speak to or link to specific changes, or remove from the update.

```suggestion
```